llvm/lib/Target/SBF/SBFISelLowering.cpp

```diff
@@ -1091,23 +1089,20 @@ SBFTargetLowering::EmitInstrWithCustomInserter(MachineInstr &MI,
   int CC = MI.getOperand(3).getImm();
   int NewCC;
   switch (CC) {
-#define SET_NEWCC(X, Y) \
-  case ISD::X: \
-    if (is32BitCmp && HasJmp32) \
-      NewCC = isSelectRROp ? SBF::Y##_rr_32 : SBF::Y##_ri_32; \
-    else \
-      NewCC = isSelectRROp ? SBF::Y##_rr : SBF::Y##_ri; \
+#define SET_NEWCC(X, Y)                                                        \
+  case ISD::X:                                                                 \
+    NewCC = isSelectRROp ? SBF::Y##_rr : SBF::Y##_ri;                          \
     break
-  SET_NEWCC(SETGT, JSGT);
-  SET_NEWCC(SETUGT, JUGT);
-  SET_NEWCC(SETGE, JSGE);
-  SET_NEWCC(SETUGE, JUGE);
-  SET_NEWCC(SETEQ, JEQ);
-  SET_NEWCC(SETNE, JNE);
-  SET_NEWCC(SETLT, JSLT);
-  SET_NEWCC(SETULT, JULT);
-  SET_NEWCC(SETLE, JSLE);
-  SET_NEWCC(SETULE, JULE);
+    SET_NEWCC(SETGT, JSGT);
```

Comment:
I think this reindentation is wrong. The macro invocations should be indented to the same level as `case` and `default` labels.

Reply:
I got this indentation from `clang-format` and thought it was right. I'll revert to the original then.